packages/components/src/resizable-box/index.tsx

```diff
@@ -124,4 +124,6 @@ function ResizableBox(
 	);
 }
 
-export default forwardRef( ResizableBox );
+const ResizableBox = forwardRef( UnforwardedResizableBox );
+
+export default ResizableBox;
```

Comment:
Tweaked the component export so we can use the TypeScript data in the Storybook props table.